pkgs/tools/misc/tvnamer/default.nix

```diff
@@ -14,6 +15,10 @@ let
           sha256 = "sha256-gTAjJpaGBF+OAeIonMHn6a5asi3dHihJqQk6s6tycOs=";
         };
 
+        nativeBuildInputs = with python3Packages; [
```

Comment:
```suggestion
        nativeBuildInputs = with python3.pkgs; [
```

Reply:
I couldn't use `super` here, as it's super coming from `overridePythonAttrs`. I did a bit of renaming, hope it's better now.